Automattic/jetpack
# Added Title attribute to the iframe for Archive.org book embed

This commit adds a `Title` attribute to the iframe for Archive.org book embed to improve the Section 508 and WCAG 2.0 accessibility compliance.


The change here tackles the open issue #7370 "Shortcode Embeds: should include title attribute on iframe output to meet accessibility standards". There are multiple files which have iframes without a title attribute, but this changes tackles the first one I found in order to establish a correct format / methodology before updating the rest. 
This change adds a fixed text string which would be a translatable string within Jetpack's i8ln language strings. However, as this embed is generated from a shortcode it would also be an option to enable this string to be specified by the user as an additional attribute to the shortcode.

Fixes #

#### Changes proposed in this Pull Request:

A title attribute with a fixed i8ln string is added to the iframe that is rendered within this piece of code.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
This adds to an existing feature of Jetpack.

#### Testing instructions:
To test this please try adding an Archive.org book using the `archiveorg-book` shortcode. E.g. 

`[archiveorg-book id=inferno00dant_2 width=560 height=384]`

With the code in place the rendered iframe should now have a title attribute.

#### Proposed changelog entry for your changes:
This commit adds a `Title` attribute to the iframe for Archive.org book embed to improve the Section 508 and WCAG 2.0 accessibility compliance.
